README.md

```diff
@@ -1,114 +1,142 @@
+[Webpack](https://webpack.js.org/), or include a global `MathBox` object onto
+your page by including the library via CDN.
 
-Open the included `/examples` to see more demos.
+### NPM Package
 
----
+- Run the following in your project's directory to install MathBox and
+  [Three.js][three-url] via [npm][npm-url]:
 
-## Docs & Help
+```bash
+npm install --save mathbox three
+```
 
-See:
+Import `THREE` and `MathBox` (library and stylesheet), along with a controls
+instance that you'll pass to the `MathBox.mathBox` constructor:
 
-- [Quick Start](docs/intro.md) for a hands on introduction.
-- [Glossary](docs/glossary.md) of terms to help get familiar with MathBox and WebGL.
-- [MathBox API](docs/api.md) for typical usage.
-- [List of Primitives](docs/primitives.md) for a full element reference.
-- [Writing Custom Shaders](docs/shaders.md) for info on custom shaders and GPU-side processing.
-- [Context API](docs/context.md) for advanced usage.
+```js
+import "mathbox/mathbox.css"
 
-Join us in the [MathBox Google Group](https://groups.google.com/forum/#!forum/mathbox) or #mathbox on Freenode.
+import { * as THREE } from "three"
+import { OrbitControls } from "three/examples/jsm/controls/OrbitControls.js"
+import { * as MathBox } from "mathbox"
+```
 
----
+### Install via CDN
+
+Include the following in your HTML header to load all required libraries and
+styles:
+
+```html
+<!-- Install your choice of three.js version from CDN: -->
+<script
+  type="text/javascript"
+  src="https://cdn.jsdelivr.net/npm/three@0.137.0/build/three.min.js"
+></script>
+
+<!-- Load a Controls instance, making sure that the version matches the Three.js version above: -->
+<script
+  type="text/javascript"
+  src="https://cdn.jsdelivr.net/npm/three@0.137.0/examples/js/controls/OrbitControls.js"
+></script>
+
+<!-- Install the latest MathBox, either mathbox.js or mathbos.min.js -->
+<script
+  type="text/javascript"
+  src="https://cdn.jsdelivr.net/npm/mathbox@latest/build/bundle/mathbox.js"
+></script>
+
+<!-- Include the MathBox CSS: -->
+<link
+  rel="stylesheet"
+  href="https://cdn.jsdelivr.net/npm/mathbox@latest/build/mathbox.css"
+/>
+```
 
 ## Basic Usage
 
-Construct a MathBox instance using the provided `mathBox()` constructor:
-
-```javascript
-import { mathBox } from "mathbox";
-import "mathbox/mathbox.css";
-
-const mathbox = mathBox(options);
+Construct a MathBox instance by passing initialization options to the
+`mathBox()` constructor:
 
-const three = mathbox.three;
-// three.renderer, three.scene, three.camera
+```js
+const options = {
+  controls: {
+    // Orbit controls, i.e. Euler angles, with gimbal lock
+    klass: THREE.OrbitControls
+  },
+};
+const root = MathBox.mathBox(options);
 ```
 
-See [threestrap](https://github.com/unconed/threestrap) for all available `options`. e.g. To spawn inside a specific element, do:
+> See [threestrap](https://github.com/unconed/threestrap) for all available
```

Comment:
~I don't think this needs to be a block quote.~ Edit: Seems like a reasonably common way to highlight stuff on github.

Reply:
Today I learn https://stackoverflow.com/a/72327818/2747370

> **Note**
> This is a note

_Not necessarily suggesting adding the note. Just...sharing weird stuff I learn about github's markdown._

Reply:
that is great! These callouts are one of the reasons I've used asciidoc in other projects. I'll make this change...